app/seeders/20190606083218-demo-field-of-activity.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.bulkInsert('fields_of_activity', [
+  {
+    id: 1,
+    name: 'Test Name',
```

Comment:
You can use something that looks like a field of activity. It will be more helpful when testing.